src/Common/nunit/ConsoleOptions.cs

```diff
@@ -382,8 +382,10 @@ private void ConfigureOptions()
             this.Add("trace=", "Set internal trace {LEVEL}.\nValues: Off, Error, Warning, Info, Verbose (Debug)",
                 v => InternalTraceLevel = RequiredValue(v, "--trace", "Off", "Error", "Warning", "Info", "Verbose", "Debug"));
 
+#if !NETCF
             this.Add("teamcity", "Turns on use of TeamCity service messages.",
                 v => TeamCity = v != null);
+#endif
```

Comment:
I do not think that any of the other properties are currently removed for the NUnitLite build, even if the option is removed.

 

From: CharliePoole [mailto:notifications@github.com] 
Sent: Monday, April 27, 2015 6:16 AM
To: nunit/nunit
Cc: Neil Colvin
Subject: Re: [nunit] Issue #611: Remove unused --teamcity option from CF build of NUnitLite (#619)

 

In src/Common/nunit/ConsoleOptions.cs <https://github.com/nunit/nunit/pull/619#discussion_r29114837> :

>              this.Add("teamcity", "Turns on use of TeamCity service messages.",
>                  v => TeamCity = v != null);
> +#endif

Shouldn't the property be removed as well?

—
Reply to this email directly or view it on GitHub <https://github.com/nunit/nunit/pull/619/files#r29114837> .  <https://github.com/notifications/beacon/AFh64YhCwPqgNykq_0j_uXzgHHRnWzDlks5oDVsCgaJpZM4EJMBn.gif>